### PR TITLE
dev-python/pandas-datareader: version bump 0.8.1

### PR DESCRIPTION
**Pull request by TheChymera**
Package-Manager: Portage-2.3.83, Repoman-2.3.20
Signed-off-by: Horea Christian <chr@chymera.eu>

### CODE REVIEW

File: dev-python/pandas-datareader/pandas-datareader-0.8.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+RDEPEND="
+	dev-python/lxml[${PYTHON_USEDEP}]
+	>=dev-python/pandas-0.19.2[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	dev-python/wrapt[${PYTHON_USEDEP}]
+	doc? (
+		dev-python/ipython[${PYTHON_USEDEP}]
+		dev-python/matplotlib[${PYTHON_USEDEP}]
+		dev-python/sphinx[${PYTHON_USEDEP}]
```
Comment (juippis) [inline dev-python/pandas-datareader/pandas-datareader-0.8.1.ebuild]: Could you try using `distutils_enable_sphinx ` here?
https://devmanual.gentoo.org/eclass-reference/distutils-r1.eclass/index.html

File: dev-python/pandas-datareader/pandas-datareader-0.8.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		dev-python/sphinx_rtd_theme[${PYTHON_USEDEP}]
+		)
+	"
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	test? (
+		${RDEPEND}
+		dev-python/pytest[${PYTHON_USEDEP}]
```
Comment (juippis) [inline dev-python/pandas-datareader/pandas-datareader-0.8.1.ebuild]: And `distutils_enable_tests ` here?

File: dev-python/pandas-datareader/pandas-datareader-0.8.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 )
```
Comment (juippis) [inline dev-python/pandas-datareader/pandas-datareader-0.8.1.ebuild]: Any chance to test on later implementations?

File: dev-python/pandas-datareader/pandas-datareader-0.8.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+	dev-python/wrapt[${PYTHON_USEDEP}]
+	doc? (
+		dev-python/ipython[${PYTHON_USEDEP}]
+		dev-python/matplotlib[${PYTHON_USEDEP}]
+		dev-python/sphinx[${PYTHON_USEDEP}]
+		dev-python/sphinx_rtd_theme[${PYTHON_USEDEP}]
+		)
+	"
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
```
Comment (juippis) [inline dev-python/pandas-datareader/pandas-datareader-0.8.1.ebuild]: Most likely wrong, and by default handled by *distutils-r1.eclass*.

**Comment by TheChymera** [comment]
@juippis ok

**Pull request closed without merging**